src/rest_api/class-openagenda-controller.php

```diff
@@ -895,15 +904,15 @@ public function submit_item( $request ) {
 	 * @param string $base64_file The base64 file.
 	 * @param string $title The title of the image.
 	 *
-	 * @return array|bool The attachment ID and URL.
+	 * @return array|\WP_Error The attachment ID and URL.
 	 */
 	public function save_file( $type, $base64_file, $title ) {
 
 		// Upload dir.
 		$upload_dir  = wp_upload_dir();
 		$upload_path = str_replace( '/', DIRECTORY_SEPARATOR, $upload_dir['path'] ) . DIRECTORY_SEPARATOR;
 
-		if ( 'data:' === substr( $base64_file, 0, 5 ) ) {
+		if ( str_starts_with( $base64_file, 'data:' ) ) {
```

Comment:
Ik weet dat phpStorm deze wijziging als suggestie geeft. Maar daarmee zorg je er wel voor dat de plugin alleen met PHP 8 gebruikt kan worden. Pas vanaf 8 is de functie `str_starts_with` beschikbaar, daarom heb ik er bewust voor gekozen die niet te gebruiken.

Reply:
Ah ok! Ik heb het teruggezet en daarnaast op andere plekken ook str_starts_with vervangen door substr. Kun je nogmaals de PR controleren?